packages/ledger-amino/src/launchpadledger.ts

```diff
@@ -166,6 +167,17 @@ export class LaunchpadLedger {
     await this.verifyCosmosAppIsOpen();
   }
 
+  public async verifyAddress(hdPath: HdPath): Promise<AddressAndPublicKeyResponse | ErrorResponse> {
+    await this.verifyDeviceIsReady();
+    assert(this.app, `${this.ledgerAppName} Ledger App is not connected`);
+
+    const hdPathToUse = hdPath;
```

Comment:
As in the other methods, `hdPath` should be optional for consistency because the `LaunchpadLedger` instance already has paths configured.

There might be a good reason to remove the default, but then we should do it for all methods.